### PR TITLE
[ALLUXIO-3358] In some cases,DefaultFileSystemMaster#listStatus throws 'Path is no longer valid, possibly due to a concurrent delete.'

### PR DESCRIPTION
**Pull request by witgo**
https://alluxio.atlassian.net/browse/ALLUXIO-3358

### CODE REVIEW

**Comment by aaudiber** [comment]
Thanks @witgo! Is it possible to add a test case that fails without the fix?

File: core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java [hunk 1]
```diff
@@ -979,6 +979,9 @@ private void listStatusInternal(LockedInodePath currInodePath, AuditContext audi
             InodeTree.LockMode.READ, child, childComponents)) {
           listStatusInternal(childInodePath, auditContext,
               nextDescendantType, statusList);
+        } catch (InvalidPathException e) {
+          LOG.warn("an invalid ptah was list, parent directory {}",
```
Comment (aaudiber) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: Can we move this to `debug` level? More information about Alluxio logging conventions: https://www.alluxio.org/docs/master/en/contributor/Code-Conventions.html#logging-conventions

Comment (aaudiber) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: minor: Capitalize first letter of log message

Comment (aaudiber) [inline core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java]: Should we list the child path as well?

**Comment by witgo** [comment]
@aaudiber  done.

**Comment by witgo** [comment]
@gpang  done

**Pull request merged**